serveradmin/dataset/commit.py

```diff
@@ -354,26 +375,18 @@ def commit_changes(
         )
 
 
-def _log_changes(deleted_servers, changed_servers, app, user):
-    # Import here to break cyclic import
-    from serveradmin.dataset import query, filters
-
-    if deleted_servers:
-        old_servers = list(query(object_id=filters.Any(*deleted_servers)))
-    else:
-        old_servers = []
-
-    if not (changed_servers or old_servers):
+def _log_changes(deleted_servers, changes, app, user):
```

Comment:
I am renaming them to `servers_to_change` and `servers_to_delete` as you suggested above.